src/debug.js

```diff
@@ -187,4 +187,39 @@ Debug.prototype.getLogger = function (name) {
   return logger;
 };
 
+/**
+ * Create a synchronous 'getLogger' method that binds around an asynchronous
+ * logger by creating a buffer until the asynchronous logger is resolved.
+ * @see {ModuleInternal.loadLinks} for where this method is bound to the created
+ *     external interface.
+ * @method getLoggingShim
+ * @param {Function} asyncMethod The wrapper to 'getLogger'
+ */
+Debug.prototype.getLoggingShim = function (asyncMethod) {
+  return function getLogggerSync(name) {
```

Comment:
Problem is that this returns the function 'getLoggerSync'.  Wanted to name
them differently to prevent confusion of multiple functions with the same
name.

On Thu, Mar 26, 2015 at 11:09 PM, Raymond Cheng <notifications@github.com>
wrote:

> In src/debug.js
> <https://github.com/freedomjs/freedom/pull/249#discussion_r27277649>:
>
> > @@ -187,4 +187,39 @@ Debug.prototype.getLogger = function (name) {
> >    return logger;
> >  };
> >
> > +/**
> > + * Create a synchronous 'getLogger' method that binds around an asynchronous
> > + * logger by creating a buffer until the asynchronous logger is resolved.
> > + * @see {ModuleInternal.loadLinks} for where this method is bound to the created
> > + *     external interface.
> > + * @method getLoggingShim
> > + * @param {Function} asyncMethod The wrapper to 'getLogger'
> > + */
> > +Debug.prototype.getLoggingShim = function (asyncMethod) {
> > +  return function getLogggerSync(name) {
>
> getLogggerSync?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freedomjs/freedom/pull/249/files#r27277649>.
>